networkoperator/v1/types_egressrouter.go

```diff
@@ -247,6 +247,8 @@ type EgressRouterStatusCondition struct {
 type EgressRouterStatus struct {
 	// Observed status of the egress router
 	// +required
+	// +listType=map
+	// +listMapKey=type
 	Conditions []EgressRouterStatusCondition `json:"conditions,omitempty" protobuf:"bytes,1,rep,name=conditions"`
```

Comment:
looks like this has protobuf, does this need the patch tags?

Reply:
Has protobuf, but is actually a CRD, I checked this against a running cluster yesterday